xml/System.Threading.Tasks/Task.xml

```diff
@@ -2599,7 +2599,7 @@ Task t Status: RanToCompletion, Result: 42
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Tasks that throw unhandled exceptions store the resulting exception and propagate it wrapped in a <xref:System.AggregateException> in calls to <xref:System.Threading.Tasks.Task.Wait%2A> or in accesses to the <xref:System.Threading.Tasks.Task.Exception%2A> property. Any exceptions not observed by the time the task instance is garbage collected will be propagated on the finalizer thread. For more information and an example, see [Exception Handling (Task Parallel Library)](~/docs/standard/parallel-programming/exception-handling-task-parallel-library.md).  
+ Tasks that throw unhandled exceptions store the resulting exception and propagate it wrapped in a <xref:System.AggregateException> in calls to <xref:System.Threading.Tasks.Task.Wait%2A> or in accesses to the <xref:System.Threading.Tasks.Task.Exception%2A> property. On .NET 4.0, any exceptions not observed by the time the task instance is garbage collected will be propagated on the finalizer thread, which crashes the process. On .NET 4.5 and later, unobserved exceptions are not rethrown from the Finalizer. For more information and an example, see [Exception Handling (Task Parallel Library)](~/docs/standard/parallel-programming/exception-handling-task-parallel-library.md).  
```

Comment:
> are not rethrown from the Finalizer

At least not by default:
https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/throwunobservedtaskexceptions-element
Effectively .NET 4.5 changed the default value of that from true to false.

Reply:
Do we need to document that here, or is it enough that it's documented where that xml config is documented and we can describe default behavior here?

Reply:
There's already a link at the end of this line to a topic all about exceptions where the details are given.

Reply:
I think we should at least add "by default" when describing the new behavior.

Reply:
It's only the 'default' on .NET Framework, right? .NET Core doesn't offer the switch?

Reply:
Correct.  We're just explicitly listing .NET Framework versions here.

Reply:
Fixed. What do you think?